R/kernel_constructors.R

```diff
@@ -50,101 +48,137 @@ NULL
 
 #' @rdname kernels
 #' @export
-bias <- function (variance, dim = 1) {
+bias <- function (variance) {
   greta_kernel("bias",
-               gpflow_name = "Bias",
-               parameters = list(variance = variance),
-               dim = dim)
+               tf_name = "tf_bias",
+               parameters = list(variance = variance))
 }
 
 #' @rdname kernels
 #' @export
-white <- function (variance, dim = 1) {
-  greta_kernel("white",
-               gpflow_name = "White",
-               parameters = list(variance = variance),
-               dim = dim)
+constant <- function (variance) {
```

Comment:
I see you've got both bias and constant doing the same thing. Was that deliberate? If so, we should mention in the docs that bias and constant are synonymous.

Reply:
Added a note to the docs. Copied the use of both from gpflow.